ports/unix/mpconfigport.h

```diff
@@ -223,10 +223,11 @@ extern const struct _mp_obj_module_t mp_module_jni;
     MICROPY_PY_JNI_DEF \
     MICROPY_PY_UTIME_DEF \
     MICROPY_PY_SOCKET_DEF \
-    { MP_ROM_QSTR(MP_QSTR_umachine), MP_ROM_PTR(&mp_module_machine) }, \
+    MP_BUILTIN_MODULE(machine), \
```

Comment:
Technically I've broken compatibility here. But the module should really be named `machine`. What should we do about it?

Since I don't think it has many users, I think we can just break it.

Reply:
Nice guess, but nope, that won't work. Please learn how "machine" module is implemented for Unix port.

Reply:
Okay, so I've found 2eb844e0b4e649034115dd6ca77dcdecd92ef547. In that case I'd say the preferable solution is as was done with `_thread` or `_webrepl`, since the `u` prefix is to indicate "that's a partial impl of a Python module for MicroPython", as per the docs `To ease extensibility, MicroPython versions of standard Python modules usually have u (“micro”) prefix.` And there's no `machine` module in CPython.

So maybe it should be named `_machine`?

Reply:
> So maybe it should be named `_machine`?

If you think ahead, you'll follow along the path of that decision. Per CPython convention, "_" modules are not intended for consumption by users, per MicroPython conventions, "u" modules are intended for such consumtion. In this regard, user may (at their own risk, as there's no such module in other ports), use "umachine" with unix port without installing full "machine" from micropython-lib.

Reply:
I'll just undo this, irrelevant to this PR.

Reply:
Undone it now, I seem to have forgotten this bit